deploy/.env

```diff
@@ -0,0 +1,3 @@
+GITHUB_OWNER=avimoondra
+GITHUB_REPO=fishbowl
+GITHUB_TOKEN=SECRET # personal access token with public_repo scope
```

Comment:
@avimoondra, this is [set](https://dashboard.render.com/static/srv-braopqqsn2vubssajk3g/env) to a `public_repo` PAT I created for this purpose, but it might make sense for you to create one?

Reply:
Created one, and updated here: https://dashboard.render.com/static/srv-braopqqsn2vubssajk3g/env